### ISSUE TITLE
Doesn't work with GitHub Organisations

### ISSUE THREAD

**Issue by flipback** [SystangoTechnologies/commit-message-lint#2]
First of all, thank you for the useful app! 

I've tried to use the app in my personal repo, and it works perfectly. However, I'm stuck with using it in a repo, which belongs to an organization. [Here](https://github.com/ASNeG/OpcUaWebServer/pull/34) is the PR.

If you look at the [details](https://github.com/ASNeG/OpcUaWebServer/pull/34/checks?check_run_id=163205888), you can see that the checker is queued but will never run.

**Comment by sumits-systango** [comment]
Hi @flipback 
Thanks for the feedback. We are looking into it and will get back to you.
Please note, we will be pushing an update to this app soon to allow usage of this app on private repos (although I know yours is not private).

**Comment by pallav-systango** [comment]
@flipback , we fixed this issue. Please try now

**Comment by flipback** [comment]
Still not working =( https://github.com/ASNeG/OpcUaWebServer/pull/34/checks?check_run_id=164790872

PS. I've added new access requirements for the app.

**Comment by sumits-systango** [comment]
I checked with a public and private repo inside an organisation and it's working fine. Can you check your CI system? Maybe it's queued and waiting on some other task to finish?

**Comment by flipback** [comment]
I've created a new PR without CI:

https://github.com/ASNeG/OpcUaWebServer/pull/35/checks?check_run_id=164835210

It isn't working.

**Comment by sumits-systango** [comment]
We'll take further look into it. Have a look at the configuration and setup in the meanwhile:
https://github.com/SystangoTechnologies/commit-message-lint/wiki/Commit-Message-Lint---2.0

**Comment by flipback** [comment]
That's strange but the app works with other repos in the organization:

https://github.com/ASNeG/OpcUaStack/pull/266/checks?check_run_id=165409273
